src/app/components/Curation/index.tsx

```diff
@@ -71,6 +73,21 @@ export default ({
   switch (componentName) {
     case NOT_SUPPORTED:
       return null;
+    case BILLBOARD:
```

Comment:
As above, let's also remove this - this ticket is only for a standalone Billboard component which is visible in Storybook